test/controllers/beneficial.owner.gov.controller.spec.ts

```diff
@@ -1231,7 +1231,8 @@ describe("BENEFICIAL OWNER GOV controller", () => {
       expect(mockSaveAndContinue).toHaveBeenCalledTimes(1);
     });
 
-    test("catch error when removing data", async () => {
+    // ASM-350 - need fix to create a real error
```

Comment:
Can this be fixed as part of this PR to avoid skipping tests?

Reply:
We need to discuss these tests because they are there just to make crash and get a 500 page
I don't know if they are really necessary, if we need to spend time on them

Reply:
I have just realised that my answer was not saved.

We need to discuss this because these tests are only there to check that the application crashes and returns a 500 error page.

Reply:
skipped tests pass